pkgs/jnigen/java/src/main/java/com/github/dart_lang/jnigen/apisummarizer/disasm/TypeUtils.java

```diff
@@ -66,13 +65,16 @@ public static Set<String> access(int access) {
   private static final Map<String, Integer> acc = new HashMap<>();
 
   static {
+    // TODO(#1079): Once we're no longer using doclet, send the bitmask instead.
```

Comment:
nit: It's better to put the full issue URL, rather than the number.

Reply:
Done. Also created an issue to do it everywhere: https://github.com/dart-lang/native/issues/1522

Probably there can be some lint for it + quickfix.